bundle/metadata/dependencies.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- type: olm.package
+  value:
+    packageName: ocs-operator
+    version: ">=4.7.0 <=4.9.0"
```

Comment:
I am little worried about this line. Will this be able to pull a version like `4.9.0-2.ci`?

Reply:
YOu can read more about versions here https://olm.operatorframework.io/docs/concepts/olm-architecture/dependency-resolution/#declaring-dependencies

Reply:
Hmm, I am still wondering. Could it make more sense to have something like `>=4.9.0 < 4.10.0` there? This could try to use ocs-operator from 4.8 or 4.7. Is that what we really want?

Reply:
Done